benchmarks/bench_batch_attention.py

```
@@ -52,110 +194,284 @@ def run_bench(
+    persistent_plan()  # warmup module loading
+    start_time = time.perf_counter()
+    persistent_plan()
+    end_time = time.perf_counter()
+    measurements_new_normal = bench_gpu_time(
+        lambda: wrapper.run(q, kv_data),
+        repeat_iters=repeats,
+    )
+    ms_new_normal = (
+        np.mean(measurements_new_normal) + (end_time - start_time) * 1000 / NUM_LAYERS
+    )
+    o, _ = wrapper.run(q, kv_data)
```

Ruff (0.14.0)

Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)